modules/src/ics04_channel/context.rs

```diff
@@ -154,6 +154,33 @@ pub trait ChannelKeeper {
                     }
                 }
             }
+            PacketResult::Timeout(res) => {
+                match res.channel {
+                    Some(c) => {
+                        //Ordered Channel
+                        let mut channel = c;
+                        channel.state = State::Closed;
+                        self.store_channel(
+                            (res.port_id.clone(), res.channel_id.clone()),
+                            &channel,
+                        )?;
+
+                        self.delete_packet_commitment((
+                            res.port_id.clone(),
+                            res.channel_id.clone(),
+                            res.seq,
+                        ))?;
+                    }
+                    None => {
+                        //Unordered Channel
+                        self.delete_packet_commitment((
+                            res.port_id.clone(),
+                            res.channel_id.clone(),
+                            res.seq,
+                        ))?;
```

Comment:
Could we refactor this so that we have a single call to `self.delete_packet_commitment` ? You can entirely delete the `None` match arm, and put the call to `delete_packet_commitment` after matching on `res.channel`.